<ui version="4.0" >
 <class>OmgLayerSelectorBase</class>
 <widget class="QDialog" name="OmgLayerSelectorBase" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Raster Layer Selector</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/resources.qrc" >
    <normaloff>:/om_icon_small.png</normaloff>:/om_icon_small.png</iconset>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="4" >
    <widget class="QLabel" name="lblBaseDir" >
     <property name="text" >
      <string>Base directory</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8" >
    <widget class="QTreeWidget" name="treeFiles" >
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <property name="columnCount" >
      <number>1</number>
     </property>
     <column>
      <property name="text" >
       <string>File</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" colspan="8" >
    <widget class="QTextBrowser" name="tbNotes" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label" >
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources/resources.qrc" >:/projectionOk.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>331</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="6" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../resources/resources.qrc" >:/projectionError.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>File with projection info</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" colspan="5" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>File missing projection info</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7" >
    <widget class="QToolButton" name="toolSupportedFileFormats" >
     <property name="text" >
      <string/>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >
       <normaloff>:/short_help.png</normaloff>:/short_help.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="6" >
    <widget class="QToolButton" name="toolRefresh" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >
       <normaloff>:/status_restart.png</normaloff>:/status_restart.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5" >
    <widget class="QToolButton" name="toolDirectorySelector" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../resources/resources.qrc" >
       <normaloff>:/fileopen.png</normaloff>:/fileopen.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>treeFiles</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OmgLayerSelectorBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>465</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel" >
     <x>275</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OmgLayerSelectorBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>393</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
